client/payment-details/dispute-details/index.tsx

```diff
@@ -4,9 +4,28 @@
  * External dependencies
  */
 import React from 'react';
+/**
+ * Internal dependencies
+ */
+import { Dispute } from '../../types/disputes';
+import { Card, CardBody } from '@wordpress/components';
+import './style.scss';
+
+interface DisputeDetailsProps {
+	dispute: Dispute;
+}
 
-const DisputeDetails: React.FC< Record< string, never > > = () => {
-	return <></>;
+const DisputeDetails: React.FC< DisputeDetailsProps > = ( { dispute } ) => {
+	return (
+		<div>
+			<Card className="transaction-details-dispute-details-wrapper">
+				<CardBody>
+					<div></div>
+				</CardBody>
+			</Card>
+			<br />
```

Comment:
It's a bit hacky to add `<br />` here but couldn't figure out why the bottom margin does not take space without it.

<img width="1189" alt="Screenshot 2023-08-14 at 10 12 47" src="https://github.com/Automattic/woocommerce-payments/assets/73803630/e7a8ee27-d179-4096-aa2f-39b8ae3d2961">

Reply:
I tried to use just plain `<div>` and style it with border and margin but afraid that if the border color is hardcoded, it might look out of place if theme is changed, so I stick with using `<Card>` but there is that weird margin bottom problem I explained above.

What do you think @Automattic/helix ?

Reply:
Agree that `<br />` is hacky. Maybe we merge now and review when working on #6927 ?

Reply:
Sounds good to me.